VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-quick-link/copy-quick-link.component.spec.ts

```diff
@@ -47,13 +47,15 @@ describe('CopyQuickLinkComponent', () => {
     });
 
     it('renders icon to copy to clipboard', () => {
+        component.ngOnInit();
```

Comment:
Could you check if this causes the Id to have 2 ids?

Reply:
Would this be valid?
![image](https://user-images.githubusercontent.com/80695759/134514950-9c263dd2-dc8f-474a-9329-e00bc622bd49.png)